int16 setpoint
int16 state_red
int16 state_green
int16 count_red
int16 count_green
